build.py

```diff
@@ -86,17 +86,26 @@ def install_llvm_tool(name, source_location, prefix, debug, jobs=1, clean=True,
         Return absolute path to executable on success and terminate with error on failure
     '''
     release = 'release_40'
-    prefix += '/llvm-4.0'
+    if not os.path.isdir(prefix): os.makedirs(prefix)
 
-    git_checkout = '( git checkout {0} && git reset --hard {0} )'.format(release) if clean else 'git checkout {}'.format(release)
+    llvm_version='4.0.0'
+    prefix += '/llvm-' + llvm_version
+    clang_path = "{prefix}/tools/clang".format(**locals())
 
-    if not os.path.isdir(prefix): os.makedirs(prefix)
 
-    if not os.path.isdir(prefix+'/.git'): execute('Clonning llvm...', 'cd {} && git clone https://github.com/llvm-mirror/llvm.git .'.format(prefix) )
-    execute('Checking out LLVM revision: {}...'.format(release), 'cd {prefix} && ( {git_checkout} || ( git fetch && {git_checkout} ) )'.format(prefix=prefix, git_checkout=git_checkout) )
+    if not os.path.isdir(prefix):
```

Comment:
Before i explicitly checked for `/.git` dir instead of just for prefix to exists. Reason for this is that if network connection hang and user terminate the script then dir will be created but it will be empty, so by checking that some content is exist inside the prefix we can avoid recovery by-hands in sub sequential runs. Therefor: could you please change line above to something like `if not os.path.isfile(prefix + '/CMakeLists.txt')`?